ansible/vars/platform_configs/upp-d-prov-eu.yaml

```diff
@@ -0,0 +1,23 @@
+debug_key_name: upp-k8s-provisioning-debug
```

Comment:
I'm constantly using this minimal template to test the provisioner modifications, so I thought it would be useful for us to have it versioned.